packages/e2e/package.json

```diff
@@ -32,10 +33,11 @@
       "@saltcorn/data/(.*)": "@saltcorn/data/dist/$1",
       "@saltcorn/types/(.*)": "@saltcorn/types/dist/$1",
       "@saltcorn/markup$": "@saltcorn/markup/dist",
-      "@saltcorn/markup/(.*)": "@saltcorn/markup/dist/$1"
+      "@saltcorn/markup/(.*)": "@saltcorn/markup/dist/$1",
+      "@saltcorn/models-common/(.*)": "@saltcorn/models-common/dist/$1"
```

Comment:
Could we name the new package @saltcorn/admin-models - I think this is a better name, models only required by the admin interface

You can probably just get away with doing a find/replace in the code base

Reply:
Okay, I renamed it